ftl/node/builder.py

```diff
@@ -85,5 +90,17 @@ def CreatePackageBase(self, base_image, cache):
             return dep_image
 
 
+def check_gcp_build(package_json, app_dir):
+    scripts = package_json.get('scripts', {})
+    gcp_build = scripts.get('gcp-build')
+
+    if not gcp_build:
+        return
+
+    os.environ["NODE_ENV"] = "development"
```

Comment:
We probably need to drop the os.environ after we finish these two. You could either grab the current environment as a dictionary, set NODE_ENV, and pass it into check_call (so it only applies to these calls), or add a try: finally clause here to make sure it gets unset after.